scripts/provision.sh

```diff
@@ -208,7 +236,7 @@ generate_bldr_keys() {
 }
 
 upload_ssl_certificate() {
-  if [ ${APP_SSL_ENABLED} = true ]; then
+  if [ "${APP_SSL_ENABLED}" = "true" ]; then
```

Comment:
Should this also be `==`?

Reply:
Nope. As I understand it, the `[]` in bash are a synonym for `test` and for the `test` syntax this is correct.